pdb.py

```diff
@@ -312,8 +312,11 @@ def _init_pygments(self):
                          if self.config.use_terminal256formatter
                             and '256color' in os.environ.get('TERM', '')
                          else TerminalFormatter)
-            self._fmt = Formatter(bg=self.config.bg,
-                                  colorscheme=self.config.colorscheme)
+            if self.config.use_terminal256formatter:
+                self._fmt = Formatter(style=self.config.style)
+            else:
+                self._fmt = Formatter(bg=self.config.bg,
+                                      colorscheme=self.config.colorscheme)
```

Comment:
Ah, so `style` is only for the `Terminal256Formatter` (had to look that up in the code though).
What about setting the formatter itself in the first place, which is already supported?

The current approach breaks if you would use the 256-color formatter, but `style` is not configured.
Maybe it should force the 256-color formatter if `style` is configured?

(just did a PR in this area myself - https://github.com/antocuni/pdb/pull/112)